### ISSUE TITLE
ESM build is empty

### ISSUE THREAD

**Issue by drwpow** [duetds/date-picker#59]
**Describe the bug**

`@duetds/date-picker`’s ESM build is currently broken.

**To Reproduce**

If you inspect the `package.json`, the `"module"` entrypoint is set to `./dist/index.js`. This is [its contents](https://unpkg.com/browse/@duetds/date-picker@1.1.0/dist/index.js):

```js
export * from './esm-es5/index.js';
```

If you navigate to [`./esm-es5/index.js`](https://unpkg.com/browse/@duetds/date-picker@1.1.0/dist/esm-es5/index.js), though, it is **blank**. As a result, the ESM build of `@duetds/date-picker` is empty.

I made a [reproduction using Snowpack](https://github.com/drwpow/snowpack-duetds), an ESM installer.

**Expected behavior**
The `"module"` entrypoint of the package should export the expected code.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Any
 - Browser: Any
 - Version: Any

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Any
 - Browser: Any
 - Version: Any

**Additional context**

Not a part of this issue at all, but it’d also be great if `@duetds/date-picker` could add [an export map](https://docs.skypack.dev/package-authors/package-checks#export-map) as part of its package contents. This would help the ESM be further-optimized.

**Comment by WickyNilliams** [comment]
Good catch. I think this should actually be pointing to `custom-element/index.js`. I'm not familiar with export maps - is this a snowpack-specific thing, or a standard?

**Comment by drwpow** [comment]
It’s a new standard, for npm/Node. You can read more about it [on the Node docs](https://nodejs.org/api/packages.html#packages_package_entry_points). But for real-world examples check out the `package.json` in recent versions of `preact` and `svelte`.

It has a number of uses, including:
- being able to conditionally serve both ESM and CJS code to different targets (better than the `module` entrypoint, which was never officially supported)
- encapsulating your library, so users can only import “public” methods (it also is almost a form of documentation)
- makes it more efficient for bundlers to optimize delivery

**Comment by WickyNilliams** [comment]
OK, sounds like it is worthwhile. I will take a closer look!

**Comment by chardin1** [comment]
Any movement on this issue?
